src/network/serverpackethandler.cpp

```diff
@@ -370,18 +372,17 @@ void Server::handleCommand_ClientReady(NetworkPacket* pkt)
 	PlayerSAO* playersao = StageTwoClientInit(peer_id);
 
 	if (playersao == NULL) {
-		actionstream
-			<< "TOSERVER_CLIENT_READY stage 2 client init failed for peer_id: "
-			<< peer_id << std::endl;
+		actionstream <<
+			"TOSERVER_CLIENT_READY stage 2 client init failed for peer_id: " <<
+			peer_id << std::endl;
```

Comment:
better like this for consistency, also `actionstream` was wrong
```suggestion
		errorstream << "TOSERVER_CLIENT_READY stage 2 client init failed "
			"peer_id=" << peer_id << std::endl;
```